### DISCUSSION TITLE
Solved 2015 day 22 with your approach from 2021 day 21

### DISCUSSION THREAD

**Discussion by morberg** [mebeim/aoc#15] [Show and tell]
First of all, huge thanks for your effort. You provide very nice walkthroughs and code.

2021 was my first AoC year and now I'm going back to previous years. Thanks to your explanation for 2021 day 23 I managed to solve 2015 day 22 with a similar approach.

https://gist.github.com/morberg/a2ddd0a8392ea9f990192bca5b1d110e

Btw, do you have any plans to provide walkthroughs for 2015-2017?

**Comment by mebeim** [comment]
Nice one! Thank you for the kind words, I'm glad you found my walkthrough helpful. Indeed a lot of concepts and approaches can be useful for different puzzles over the years.

> Btw, do you have any plans to provide walkthroughs for 2015-2017?

I currently do not plan to do that. As interesting and fun as it might be, writing walkthroughs takes a lot of time, and I don't have much free time to dedicate to this lately. It's still something that I would like to do in the future if possible; it would be cool to have walkthroughs for all the years from 2015 onwards. Maybe some day I'll find the time (and the will) to slowly work my way through old puzzles and write walkthroughs as I solve them, who knows.
